popup.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
Santiago's changes shouldn't be in your changes. I think that you have made a mistake.

Reply:
I merged the updated changes from the main branch into mine, is that what you mean by rebase? Or do you mean call git rebase and add my changes to that? Because when I tried to use git rebase all of my changes were lost. Is there a different approach that you would recommend?